packages/client/src/v2-events/features/events/actions/declare/Review.stories.tsx

```diff
@@ -103,9 +112,9 @@ export const ReviewForLocalRegistrarComplete: Story = {
   play: async ({ canvasElement, step }) => {
     await step('Modal has scope based content', async () => {
       const canvas = within(canvasElement)
-      await userEvent.click(
-        await canvas.findByRole('button', { name: 'Register' })
-      )
+      const button = await canvas.findByRole('button', { name: 'Register' })
```

Comment:
This is to reduce flakiness in the tests as sometimes the review page does get initialised a millisecond earlier than we're able to fetch the event data and drafts to its state